test/modules/standard/gmp/ldelaney/gmp_BigInt_arithmetic.chpl

```diff
@@ -0,0 +1,40 @@
+use GMP;
+
+// test the arithmetic functions
```

Comment:
For face-to-face

I need to think more about the implications of these examples.  Is this really the API we want?
Was there other discussion about this vs. the alternatives that I am unaware of?